src/main/java/ch/njol/skript/expressions/ExprTargetedBlock.java

```diff
@@ -29,62 +30,75 @@
 import ch.njol.skript.lang.ExpressionType;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.util.Kleenean;
-import org.bukkit.Material;
 import org.bukkit.block.Block;
-import org.bukkit.entity.Player;
+import org.bukkit.entity.LivingEntity;
 import org.bukkit.event.Event;
 import org.eclipse.jdt.annotation.Nullable;
 
 @Name("Targeted Block")
-@Description("The block at the crosshair. This regards all blocks that are not air as fully solid, e.g. torches will be like a solid stone block for this expression.")
-@Examples({"# A command to set the block a player looks at to a specific type:",
-		"command /setblock &lt;material&gt;:",
-		"\ttrigger:",
-		"\t\tset targeted block to argument"})
-@Since("1.0")
-public class ExprTargetedBlock extends PropertyExpression<Player, Block> {
+@Description({
+	"The block at the crosshair. This regards all blocks that are not air as fully solid, e.g. torches will be like a solid stone block for this expression.",
+	"The actual target block will regard the actual hit box of the block."
+})
+@Examples({
+	"set target block of player to stone",
+	"set target block of player to oak_stairs[waterlogged=true]",
+	"break target block of player using player's tool",
+	"give player 1 of type of target block",
+	"teleport player to location above target block",
+	"kill all entities in radius 3 around target block of player",
+	"set {_block} to actual target block of player",
+	"break actual target block of player"
+})
+@Since("1.0, INSERT VERSION (actual/exact)")
```

Comment:
might be worth mentioning support for living entities rather than just players?
```suggestion
@Since("1.0, INSERT VERSION (support for more entities, actual/exact target block)")
```